CHANGELOG.md

```diff
@@ -11,21 +11,43 @@ Branch: `nilrt/master/hardknott`
 
 ### NILRT
```

Comment:
nitpick: NILRT -> nilrt

Also, can you fix [Unreleased] above to [9.0] ?

Reply:
Yeah; changing it to 9.0 and creating a new [unreleased] section above is fine. This PR needs to be cherry-picked into the 22.5 release branch anyway.

I expect we'll have more changelog updates to make as we go-final here. So this PR doesn't have to be perfectly complete.

Reply:
Done.